src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -160,7 +161,7 @@ class MarathonSchedulerActor private (
       val origSender = sender()
       withLockFor(appId) {
         val promise = Promise[Unit]()
-        val tasksToKill = taskIds.flatMap(taskId => taskTracker.marathonTaskSync(appId, taskId))
+        val tasksToKill = taskIds.flatMap(taskId => taskTracker.marathonTaskSync(Task.Id(taskId)))
```

Comment:
Is there a reason why KillTasks has taskids as string?
`MarathonSchedulerService.killTasks` convert from taskId->String and here back again...